edbdeploy/data/ansible/roles/bdr_pot_setup/tasks/pem_server_certs.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Gather service facts
+  service_facts:
+
+- name: Open Postgres TCP port 80
+  ansible.posix.firewalld:
+    port: "80/tcp"
+    permanent: yes
+    state: enabled
+    immediate: true
+  when:
+    - ansible_facts.services['firewalld.service'] is defined
+    - ansible_facts.services['firewalld.service'].state == 'running'
+    - ansible_facts.services['firewalld.service'].status == 'enabled'
+  become: yes
+
+- name: shutdown the httpd service
+  systemd:
+    name: httpd
+    state: stopped
+  become: yes
+
+- name: install certbot on pem server
+  package:
+    name: certbot
+    state: present
+  become: yes
+
+- name: execute certbot to getting the certificates
+  shell: |
+      set -o pipefail
+      set -eu
+      certbot certonly --standalone \
+            --preferred-challenges http \
+            -d {{ project }}pem.{{ domain }} \
+            -m {{ email_id }} \
+            --agree-tos -n
+  args:
+     executable: /bin/bash
+  become: yes
+  register: certbot_output
+  changed_when: certbot_output.rc == 0
+  failed_when: certbot_output.rc != 0
+
+- name: update the edb-ssl-pem.conf
+  lineinfile:
+     path: "/etc/httpd/conf.d/edb-ssl-pem.conf"
+     regexp: "{{ line_item.from }}"
```

Comment:
I'd suggest to use the `regex_escape` filter here:
```yaml
regexp: "{{ line_item.from | regex_escape() }}"
```

Reply:
Fixed